ecosystem/sep-0024.md

```diff
@@ -549,6 +558,7 @@ Name | Type | Description
 `kind` | string | `deposit` or `withdrawal`.
 `status` | string | Processing status of deposit/withdrawal.
 `status_eta` | number | (optional) Estimated number of seconds until a status change is expected.
+`kyc_verified` | boolean | (optional) Whether or not the anchor has verified the user's KYC information and has authorized them for this transaction.
```

Comment:
We should avoid the term authorized here, because the term has particular meaning in payments products and our overloaded use of it may cause confusion for folks in financial industries. I think its use here is unnecessary anyway and can be simply deleted.
```suggestion
`kyc_verified` | boolean | (optional) True if the anchor has verified the user's KYC information for this transaction.
```